### PR TITLE
39 fixfrontend cleanup

### LINKED ISSUE #39: fix/frontend-cleanup

### PR DESCRIPTION
**Pull request by chrsBa**

### CODE REVIEW

**Review by flyingbeat — APPROVED** [review]

File: src/app/shared/map/map.component.ts [hunk 1]
```diff
@@ -80,6 +85,25 @@ export class MapComponent {
...
+			}
+
+			if (
+				center &&
+				Math.abs(center[0] - this.lastCenterEvent[0]) > 0.1 &&
+				Math.abs(center[1] - this.lastCenterEvent[1]) > 0.1
+			) {
+				this.centerChange.emit(center);
+				this.lastCenterEvent = center;
+			}
```
Comment (flyingbeat) [inline src/app/shared/map/map.component.ts]: this does not work yet, if I only change the center (move the map without zooming) I land back on the old center, anyways I would be nice that the map shows my own marker if im guessing, and all of the markers if I already solved by default. But lets not invest too much time into it probably

File: src/app/shared/map/map.component.ts [hunk 2]
```diff
@@ -80,6 +85,25 @@ export class MapComponent {
...
+		});
 	}
 
 	public refreshMap() {
```
Comment (flyingbeat) [inline src/app/shared/map/map.component.ts]: refresh is still not working fully, If I solve a location riddle I have to refresh the app to see the guesses and solution of others. To reproduce log in with the demo user and try to solve a riddle (use the initial setup which is created when using docker compose) or see backend/infrastructure/default_state_setup/README

File: src/app/home/tabs/post/data-access/post-state.service.ts [hunk 3]
```diff
@@ -44,16 +47,21 @@ export class PostStateService {
 		)
 	);
 
+	userLocation = from(Geolocation.getCurrentPosition());
```
Comment (flyingbeat) [inline src/app/home/tabs/post/data-access/post-state.service.ts:52]: This does not always work for me, If I refresh and take a foto for the very first time there Is a blue marker with my current location (awesome!) but if I cancel and retake one the marker does not appear for any of the following pictures. If I cancel by refreshing it works (or just refresh again) probably something wrong with the cancel button

Comment (flyingbeat) [reply src/app/home/tabs/post/data-access/post-state.service.ts:52]: d0d781c5631af8692a23b9e97ae4f3501a38d32f

File: src/app/shared/map/map.component.ts (hunk 1 above)
Comment (chrsBa) [reply src/app/shared/map/map.component.ts]: https://github.com/uzh-ase-fs24/frontend/pull/44/commits/7cabe19fb17979794c5527bcd83cbc7f0b84367b

File: src/app/shared/map/map.component.ts (hunk 2 above)
Comment (chrsBa) [reply src/app/shared/map/map.component.ts]: Fix: https://github.com/uzh-ase-fs24/frontend/pull/44/commits/17f4887b1be363e779803076b74bdda161151723

**Pull request merged**